### DISCUSSION TITLE
I bought a domain in my vercel account but I don't see it

### DISCUSSION THREAD

**Discussion by alexdev888** [vercel/community#4867] [Account Issue] [answered]
### Summary

I bought a domain in my vercel account but I don't see it in my domains list. 
I have a hobby plan, so, I'm unable to contact support directly.
What I can do in this case? 
Hope for help from the community! 

### Steps to Reproduce

1. I bought a domain in my vercel account
2. Transaction went well
3. No domains in the list
4. I checked everywhere, can't find it
5. Domain is working but pointing to 404 page of vercel system

**Comment by alexdev888** [comment, marked as answer]
Got support via official channel

**Reply by GronholmRobin** [reply]
I've got the same problem. How did you solve it? Where's the official channel? Help plz 🙏

**Reply by amyegan** [reply]
Hey @GronholmRobin. I'm guessing your account was recently updated to our new structure. Do you see the domain listed in your account settings? 

For more info check out https://github.com/vercel/community/discussions/6298

**Comment by GronholmRobin** [comment]
Oh, there they are. Thanks! 🙏Maybe, not that intuitive or expected though. Best,RobinOn Mar 29, 2024, at 14:22, Amy ***@***.***> wrote:﻿
Hey @GronholmRobin. I'm guessing your account was recently updated to our new structure. Do you see the domain listed in your account settings?
For more info check out #6298
